### PR TITLE
chore(V3template): command-and-response-js for V3

### PR DESCRIPTION
**Pull request by frankqianms**

### CODE REVIEW

File: templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json [hunk 1]
```diff
@@ -142,7 +142,7 @@
         "messageTeamMembers"
     ],
     "validDomains": [
-        "{{state.fx-resource-bot.domain}}",
-        "{{state.fx-resource-bot.domain}}"
+        "${{BOT_DOMAIN}}",
+        "${{BOT_DOMAIN}}"
```
Comment (xzf0587) [inline templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json:146]: The original template has a bug here. Remove the second line. It is duplicated.

File: templates/scenarios/command-and-response-js/.fx/teamsfx.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    # Output: following environment variable will be persisted in current environment's .env file.
+    # BOT_ID: the AAD app client id created for Azure Bot
+    # BOT_PASSWORD: the AAD app client secret created for Azure Bot
+
+  - uses: arm/deploy # Deploy given ARM templates parallelly.
+    with:
+      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}} # The AZURE_SUBSCRIPTION_ID is a built-in environment variable. TeamsFx will ask you select one subscription if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select subscription if it's empty in this case.
+      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}} # The AZURE_RESOURCE_GROUP_NAME is a built-in environment variable. TeamsFx will ask you to select or create one resource group if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select or create resource grouop if it's empty in this case.
+      templates:
+      - path: ./tab/infra/azure.bicep # Relative path to this file
```
Comment (xzf0587) [inline templates/scenarios/command-and-response-js/.fx/teamsfx.yml:16]: Command-and-response is a bot project. Use the bot arm template here and remove tab arm template.

File: templates/scenarios/command-and-response-js/bot/infra/azure.bicep [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+param webAppName string = resourceBaseName
+param location string = resourceGroup().location
+
+// Compute resources for your Web App
+resource serverfarm 'Microsoft.Web/serverfarms@2021-02-01' = {
+  kind: 'app'
+  location: location
+  name: serverfarmsName
+  sku: {
+    name: webAppSKU
```
Comment (xzf0587) [inline templates/scenarios/command-and-response-js/bot/infra/azure.bicep:23]: webAppSKU should be declared before using in bicep template.

File: templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json (hunk 1 above)
Comment (hund030) [inline templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json:146]: We only need one ${{BOT_DOMAIN}}, could you help remove the redundancy?

Comment (frankqianms) [reply templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json:145]: fixed

File: templates/scenarios/command-and-response-js/.fx/teamsfx.yml (hunk 2 above)
Comment (frankqianms) [reply templates/scenarios/command-and-response-js/.fx/teamsfx.yml]: fixed

File: templates/scenarios/command-and-response-js/bot/infra/azure.bicep (hunk 3 above)
Comment (frankqianms) [reply templates/scenarios/command-and-response-js/bot/infra/azure.bicep:24]: fixed

File: templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json (hunk 1 above)
Comment (frankqianms) [reply templates/scenarios/command-and-response-js/.fx/appPackage/manifest.template.json:145]: fixed

File: templates/scenarios/command-and-response-js/.fx/teamsfx.yml (hunk 2 above)
Comment (frankqianms) [reply templates/scenarios/command-and-response-js/.fx/teamsfx.yml]: fixed

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**